### ISSUE TITLE
Pandas reindexing error when passing multiple bands to `goes_nearesttime`

### ISSUE THREAD

**Issue by olimcc** [blaylockbk/goes2go#28]
When selecting the `ABI-L1b-Rad` product, passing a single band to `goes2go.data.goes_nearesttime` yields a non-error result, but a list of multiple bands results in `pandas.errors.InvalidIndexError: Reindexing only valid with uniquely valued Index objects`. The issue is raised on [line 612](https://github.com/blaylockbk/goes2go/blob/main/goes2go/data.py#L612).

Example to illustrate the issue:

```
import datetime
from goes2go.data import goes_nearesttime

if __name__ == '__main__':

    t = datetime.datetime(2021, 1, 1, 17, 0, 0)

    # this succeeds
    bands = [1]
    res = goes_nearesttime(
        t,
        satellite=17,
        product='ABI-L1b-Rad',
        return_as='filelist',
        bands=bands,
        download=False,
        domain='C')
    print(res)

    # this fails 
    bands = [1, 2]
    res = goes_nearesttime(
        t,
        satellite=17,
        product='ABI-L1b-Rad',
        return_as='filelist',
        bands=bands,
        download=False,
        domain='C')
    print(res)
```

Environment details:

This failed on the latest release `2022.7.15` and the current HEAD commit `8a431a2`. I am running this on an M1 Pro (Apple Silicon) machine, Python `3.9.13`. I also experienced this while running in a Docker container with Ubuntu 20.04.4 & Python `3.8.10`.

**Comment by blaylockbk** [comment]
Fixed by #29

**Comment by blaylockbk** [comment]
Thanks! @olimcc
